Systems/c172p-damage.xml

```diff
@@ -288,20 +289,14 @@
             <test logic="AND" value="0.12">
                 /velocities/airspeed-kt GT damage/limits-vne-low
                 damage/roll-moment GT 4000.0
-                wing-damage/left-wing EQ 0.0
+                wing-damage/left-wing LT 0.12
```

Comment:
Both give the same effect.
Also to be consistent with the rest of the file where the equivalent conditions are 'if damage is lower than this value, increase it to this value".

Here too, not important